CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased
+## Unreleased (2020-03-19)
```

Comment:
Usually I keep Unreleased section as it is and add new section with new version and date.

Reply:
Updated. Is that what you meant?

Reply:
Yup! Thanks 👍